### PR TITLE
Setup graphql codegen

### LINKED ISSUE #16: [Front] GraphQL環境設定
- Apollo-serverの設定をする

---
余裕があれば
- graphql-codegen (https://the-guild.dev/graphql/codegen)調べる

### LINKED ISSUE #19: [Front] Codegenの実装
-GraphQL codegen

https://the-guild.dev/graphql/codegen/docs/guides/react-vue

### PR DESCRIPTION
**Pull request by Yuriko-Otorii**
close: #19 
close: #16 

- Setup GraphQL-Codegen 
[https://the-guild.dev/graphql/codegen](url)

-Debug the CORS error
It took me a long time to solve this...
But the solution was so simple. I feel like this is the programming lol

*From now, we can use query and mutation! 🥳
I'll tell you how to use them next week!

### CODE REVIEW

File: Frontend/src/features/graphql/authentication/signup.graphql
Comment (Hikari7) [inline Frontend/src/features/graphql/authentication/signup.graphql:1]: Great job, Yuri chan! 😭 Thank you so much!

📝 note:
we might need to discuss file structure 🤔 I didn't really think about where to call query and mutation in the [previous issue](https://github.com/Yuriko-Otorii/wordly/issues/29#issue-2433510032)😂 🙏 Sorry...

This is just an idea though, how about making `graphql` folder for each component in the `feature`...?

```
.
└── feature/
    └── Home/
        └── Card/
            ├── graphql.ts ...、クエリを叩いてデータをCardにpropsで渡す(file nameどうしようw）
            └── index.tsx ... UIのCard componentを呼ぶ
```

Please let me know how you did in your previous project next week! 😉

File: Frontend/src/features/graphql/types.ts
Comment (Hikari7) [inline Frontend/src/features/graphql/types.ts:1]: 🎉

**Review by Hikari7 — APPROVED** [review]
I left a comment, but we can discuss it later. LGTM!
Thank you so much for your hard work!

File: Frontend/src/features/graphql/authentication/signup.graphql
Comment (Hikari7) [reply Frontend/src/features/graphql/authentication/signup.graphql:1]: Also, I think we can locate config file like `graphqlClient.ts` outside of the `feature` file (even `components`) 🤔

Comment (Yuriko-Otorii) [reply Frontend/src/features/graphql/authentication/signup.graphql:1]: Oh, it totally slipped my mind about folder structure😂


> file nameどうしようw

クエリ名でもいいかな？分かりやすい名前にしたつもりでは…ある…w
次のとき話そ！

**Comment by Hikari7** [comment]
[結論]

- ページごとに`graphql` フォルダを作って、その中に `query` フォルダと `mutation` フォルダを作る

**Pull request merged**